code/modules/mob/living/silicon/ai/laws.dm

```diff
@@ -21,6 +21,7 @@
 		for(var/mob/living/silicon/robot/R in connected_robots)
 			if(R.lawupdate)
 				R.lawsync()

```

Comment:
I just left it as a failsafe since this bug has cropped up and been fixed repeatedly over the years. I could move it though